tests/TestFdoSecrets.cpp

```diff
@@ -82,6 +82,11 @@ void TestFdoSecrets::testSpecialCharsInAttributeValue()
         QCOMPARE(res.count(), 1);
         QCOMPARE(res[0]->title(), QStringLiteral("titleB"));
     }
+    {
+        const auto term = Collection::attributeToTerm("testAttribute", "v|");
```

Comment:
May want to test the `|v` case as well.

Reply:
I agree that more tests are need, but i think we should implement these in TestTools. The regex builder tests need some refactoring as well. Right now they just check if the regex built matches some expected regex, but not if they actually behave correctly.